libs/elasticsearch/langchain_elasticsearch/vectorstores.py

```diff
@@ -541,42 +541,48 @@ def default_doc_builder(hit: Dict) -> Document:
 class ElasticsearchStore(VectorStore):
     """`Elasticsearch` vector store.
 
-    Example:
+     Setup:
+        Install ``langchain_elasticsearch`` and running the Elasticsearch docker container.
+
+        .. code-block:: bash
+
+            pip install -qU langchain_elasticsearch
+            docker run -p 9200:9200 -e "discovery.type=single-node" -e "xpack.security.enabled=false" -e "xpack.security.http.ssl.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:8.12.1
+
+    Key init args — indexing params:
+        index_name: str
+            Name of the index to create.
+        embedding: Embeddings
+            Embedding function to use.
+
+    Key init args — client params:
+        es_connection: Optional[Elasticsearch]
+            Pre-existing Elasticsearch connection.
+        es_url: Optional[str]
+            URL of the Elasticsearch instance to connect to.
+        es_cloud_id: Optional[str]
+            Cloud ID of the Elasticsearch instance to connect to.
+        es_user: Optional[str]
+            Username to use when connecting to Elasticsearch.
+        es_password: Optional[str]
+            Password to use when connecting to Elasticsearch.
+        es_api_key: Optional[str]
+            API key to use when connecting to Elasticsearch.
+
+    Instantiate:
         .. code-block:: python
 
-            from langchain_elasticsearch.vectorstores import ElasticsearchStore
+            from langchain_elasticsearch import ElasticsearchStore
             from langchain_openai import OpenAIEmbeddings
 
-            store = ElasticsearchStore(
-                embedding=OpenAIEmbeddings(),
-                index_name="langchain-demo",
-                es_url="http://localhost:9200"
+            vector_store = ElasticsearchStore(
+                "langchain-demo", embedding=OpenAIEmbeddings(), es_url="http://localhost:9200"
```

Comment:
any reason to remove `index_name=` on that part? Seems clearer
```suggestion
                index_name="langchain-demo",
                embedding=OpenAIEmbeddings(),
                es_url="http://localhost:9200",
```